src/man/sssd-ldap.5.xml

```diff
@@ -1778,7 +1778,17 @@
                             principal used for authentication to the directory.
                             This option can either contain the full principal (for
                             example host/myhost@EXAMPLE.COM) or just the principal name
-                            (for example host/myhost).
+			    (for example host/myhost). By default, the value is not set
+			    and the following principals are used. If none of them are
+			    found, the first principal in keytab is returned.
```

Comment:
Amit, please, mind the indentation! In those lines, you've replaced space indentation for tab indentation.

Reply:
+1